addons/web/static/src/js/view_form.js

```diff
@@ -6073,13 +6073,19 @@ instance.web.form.FieldStatus = instance.web.form.AbstractField.extend({
             'value_folded': _.find(self.selection.folded, function(i){return i[0] === self.get('value');})
         });
         self.$el.html(content);
-        var statusbar_colors = JSON.parse((self.node.attrs || {}).statusbar_colors || "{}");
-        var color = statusbar_colors[self.get('value')];
-        if (color) {
-            var $color = $.Color(color);
-            var fr = $color.lightness(0.7);
-            var to = $color.lightness(0.4);
-            self.$(".oe_active, .oe_active > .arrow span").css("background-image", 'linear-gradient(to bottom, ' + fr.toHexString() + ', ' + to.toHexString() + ')');
+        statusbar_colors_attr = '' + (self.node.attrs || {}).statusbar_colors;
```

Comment:
I agree with your reasoning, but this is OCB, where we have constraints in the actions to do. And again, the goal here is to avoid large diffs that can provoke conflicts in future synchronizations.

@StefanRijnhart @hbrunn, do you support my reasoning or do you agree in making this change?